versioned_docs/version-2.0/how_to_guides/evaluation/evaluate_llm_application.mdx

```diff
@@ -358,6 +358,44 @@ One common workflow is to fetch examples that have a certain metadata key-value
   ]}
 />
 
+## Evaluate on a dataset split
+
+:::tip Recommended Reading
+
+Before diving into this content, it might be helpful to read the [guide on creating/managing dataset splits](../datasets/manage_datasets_in_application#create-and-manage-dataset-splits).
```

Comment:
haha personal pref but i don't like "diving in" because it sound ai generated these days